bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -794,9 +799,22 @@ Full pathname to the inventory initialization control file.
 <entry id="use_fates_luh" type="logical" category="physics"
        group="clm_inparm" valid_values="" >
 If TRUE, enable use of land use harmonization (LUH) state and transition data from luh_timeseries file.
+This is enabled by default if fates_harvest_mode is set to use the raw LUH2 harvest data
 (Also, only valid for use_fates = true and is incompatible with transient runs currently.)
 </entry>
 
+<entry id="use_fates_lupft" type="logical" category="physics"
+       group="elm_inparm" valid_values="" value=".false.">
+If TRUE, enable use of fates land use x pft mapping data file.
```

Comment:
It also sets:

- NOCOMP
- LUH2
- FIXED-BioGEO

which is the primary scientific study for this work. This is settling a few things at once for convenience and also making it easier to do the primary setup.

Because it's a conglomeration setting (turning several things on at once), longer term I think it should be moved into an XML setting. Technically it could also be in a build-namelist command line option or a use-case. But, we are deprecating those last two. Leave up to you if setting them all true, but leaving use_fates_lupft to FALSE is OK or not.

Conglomeration settings should also check that the user isn't trying to set this on their own. So for example if the user try's to turn this on -- but turn one of these off individually that should fail.